tests/gocase/unit/type/tdigest/tdigest_test.go

```diff
@@ -0,0 +1,143 @@
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "1", "hahah").Err(), errMsgWrongKeyword)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "compression").Err(), errMsgWrongNumberArg)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "compression", "hahah").Err(), errMsgParseCompression)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "compression", "0").Err(), errMsgNeedToBePositive)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "compression", "-1").Err(), errMsgNeedToBePositive)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "compression", "0.1").Err(), errMsgParseCompression)
+
+		require.NoError(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key0", "compression", "1").Err())
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key0", "compression", "1").Err(), errMsgKeyAlreadyExists)
+		require.NoError(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key1", "compression", "1000").Err())
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key", "compression", "1001").Err(), errMsgMustInRange)
+	})
+
+	t.Run("tdigest.info with different arguments", func(t *testing.T) {
+		keyPrefix := "tdigest_info_"
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.INFO").Err(), errMsgWrongNumberArg)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.INFO", keyPrefix+"key", "hahah").Err(), errMsgWrongNumberArg)
+		require.ErrorContains(t, rdb.Do(ctx, "TDIGEST.INFO", keyPrefix+"key").Err(), errMsgKeyNotExist)
+		require.NoError(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key0", "compression", "1").Err())
+		{
+			rsp := rdb.Do(ctx, "TDIGEST.INFO", keyPrefix+"key0")
+			require.NoError(t, rsp.Err())
+			info := toTdigestInfo(t, rsp.Val())
+			require.EqualValues(t, 1, info.Compression)
+			require.EqualValues(t, 1*6+10, info.Capacity)
+			require.EqualValues(t, 0, info.MergedNodes)
+			require.EqualValues(t, 0, info.UnmergedNodes)
+			require.EqualValues(t, 0, info.MergedWeight)
+			require.EqualValues(t, 0, info.UnmergedWeight)
+			require.EqualValues(t, 0, info.Observations)
+			require.EqualValues(t, 0, info.TotalCompressions)
+		}
+
+		// require.NoError(t, rdb.Do(ctx, "TDIGEST.CREATE", keyPrefix+"key1", "compression", "1000").Err())
+		// rsp = rdb.Do(ctx, "TDIGEST.INFO", keyPrefix+"key1")
+		// require.NoError(t, rsp.Err())
+		// info = toTdigestInfo(t, rsp.Val())
+		// require.EqualValues(t, 1000, info.Compression)
+		// require.EqualValues(t, 1024, info.Capacity) // max is 1024
+		// require.EqualValues(t, 0, info.MergedNodes)
+		// require.EqualValues(t, 0, info.UnmergedNodes)
+		// require.EqualValues(t, 0, info.MergedWeight)
+		// require.EqualValues(t, 0, info.UnmergedWeight)
+		// require.EqualValues(t, 0, info.Observations)
+		// require.EqualValues(t, 0, info.TotalCompressions)
```

Comment:
Hmm what's the situation of this commented block?

Reply:
> Hmm what's the situation of this commented block?

Oh, my mistake! Sorry that I think I forgot to recover some parts for debug. I will double check my commit today.

Reply:
Hi @PragmaTwice ,

Thanks very much for your correction! ❤

I have recovered the commented test case and have run it in my local environment.

Best Regards,
Edward